src/wallet/rpc/transactions.cpp

```diff
@@ -839,7 +839,9 @@ RPCHelpMan rescanblockchain()
 {
     return RPCHelpMan{"rescanblockchain",
                 "\nRescan the local blockchain for wallet related transactions.\n"
-                "Note: Use \"getwalletinfo\" to query the scanning progress.\n",
+                "Note: Use \"getwalletinfo\" to query the scanning progress.\n"
+                "The rescan is significantly faster if a descriptor wallet is loaded\n"
```

Comment:
In 1f0198a3fe58d42dce9e8ddaa09dc2ba8171930e "rpc: doc: mention rescan speedup using `blockfilterindex=1` in affected wallet RPCs"

Since `rescanblockchain` operates per-wallet, this should that it is faster when used on a descriptor wallet, as saying "if a descriptor wallet is loaded" incorrectly suggests that it may be faster for a legacy wallet if a descriptor wallet was also loaded at the same time.

```suggestion
                "The rescan is significantly faster when used on a descriptor wallet\n"
```